src/main/resources/application.yml

```
@@ -49,3 +49,6 @@ redis:
   host: ${REDIS_HOST}
   port: ${REDIS_PORT}
 
+socket:
+  host: ${SOCKET_HOST}
+  port: ${SOCKET_PORT}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)